x-pack/plugins/apm/ftr_e2e/cypress/integration/power_user/settings/custom_links.spec.ts

```diff
@@ -31,8 +62,6 @@ describe('Custom links', () => {
     emptyPrompt.should('not.exist');
     cy.contains('foo');
     cy.contains('https://foo.com');
-    cy.get('[data-test-subj="editCustomLink"]').click();
-    cy.contains('Delete').click();
```

Comment:
We might still want to test the delete flow (unless we already do that elsewhere)

Reply:
... and I think it might be a good idea to delete the links afterwards anyway.